tasks/main.yml

```diff
@@ -1,28 +1,34 @@
 ---
-- name: Add Nginx Rsyslog configuration file
-  template: src="{{ item }}" dest="/etc/rsyslog.d/{{ item }}" owner=root group=root
-  with_items:
-    - 21-nginx-loggly.conf
-    - 21-rails.conf
-    - 22-loggly.conf
+- fail: msg="This role requires 'loggly.token'"
+  when: loggly.token | trim == ''
 
-- name: Create cerfiticates folder If It Does Not Exist
-  file: path=/etc/rsyslog.d/keys/ca.d state=directory
+# Only supported by Ubuntu targets - will provide support for $InputFileName wilcard paths
+# - apt_repository: repo='{{ loggly.rsylog_ppa }}'
+#- debug: msg="PPA is {{ loggly.rsylog_ppa }}"
+#  when: loggly.rsylog_ppa
 
-- name: Get Certificates From Loggly
-  get_url: url=https://logdog.loggly.com/media/loggly.com.crt dest=/etc/rsyslog.d/keys/ca.d/loggly.com.crt
+- apt: name={{ item }} update_cache=yes cache_valid_time=3600
+  with_items:
+    - rsyslog-gnutls
+    - wget
 
-- name: Get Certificates From Loggly
-  get_url: url=https://certs.secureserver.net/repository/sf_bundle.crt dest=/etc/rsyslog.d/keys/ca.d/sf_bundle.crt
+- name: Create cerfiticates folder If It Does Not Exist
+  file: path={{ loggly.certificate_dest }} state=directory
 
-- name: Concatenate Certs together
-  shell: cat sf_bundle.crt loggly.com.crt > loggly_full.crt
-  args:
-    chdir: /etc/rsyslog.d/keys/ca.d/
+- name: Install loggly certificate bundle once https://en.wikipedia.org/wiki/Server_Name_Indication is supported
+  get_url: url={{ loggly.certificate_url }} dest={{ loggly.certificate_dest }}{{ loggly.certificate_file }}
+  ignore_errors: yes
```

Comment:
doesn't work with the version of Python installed by default on the servers.  Leaving it in for when we move to a newer version of Ubuntu (we can then drop the task below this).